### ISSUE TITLE
Missing Content-MD5 header for S3 DeleteObjects

### ISSUE THREAD

**Issue by baradm100** [aws/aws-sdk-java-v2#2732]
## Describe the bug
By the [documentation](https://docs.amazonaws.cn/en_us/AmazonS3/latest/API/API_DeleteObjects.html), for the `DeleteObjects` we need to set `Content-MD5` header.
When using the SDK, we're getting the following error: `Missing required header for this request: Content-MD5`

## Expected Behavior
The `Content-MD5` should be added automatically by the SDK

## Current Behavior
The `Content-MD5` is not being added to the delete objects request

### Logs
```
2021-09-22 15:21:30,343 (main) DEBUG  s.a.a.c.i.ExecutionInterceptorChain:84 - Creating an interceptor chain that will apply interceptors in the following order: [software.amazon.awssdk.awscore.eventstream.EventStreamInitialRequestInterceptor@3c0be339, software.amazon.awssdk.awscore.interceptor.HelpfulUnknownHostExceptionInterceptor@15ca7889, software.amazon.awssdk.services.s3.internal.handlers.EnableChunkedEncodingInterceptor@7a675056, software.amazon.awssdk.services.s3.internal.handlers.DisableDoubleUrlEncodingInterceptor@d21a74c, software.amazon.awssdk.services.s3.internal.handlers.EnableTrailingChecksumInterceptor@6e509ffa, software.amazon.awssdk.services.s3.internal.handlers.CreateMultipartUploadRequestInterceptor@2898ac89, software.amazon.awssdk.services.s3.internal.handlers.GetObjectInterceptor@683dbc2c, software.amazon.awssdk.services.s3.internal.handlers.ExceptionTranslationInterceptor@68267da0, software.amazon.awssdk.services.s3.internal.handlers.AsyncChecksumValidationInterceptor@2638011, software.amazon.awssdk.services.s3.internal.handlers.EndpointAddressInterceptor@6ff29830, software.amazon.awssdk.services.s3.internal.handlers.SignerOverrideInterceptor@6a2b953e, software.amazon.awssdk.services.s3.internal.handlers.GetBucketPolicyInterceptor@313b2ea6, software.amazon.awssdk.services.s3.internal.handlers.PutObjectInterceptor@2f465398, software.amazon.awssdk.services.s3.internal.handlers.SyncChecksumValidationInterceptor@548e6d58, software.amazon.awssdk.services.s3.internal.handlers.DecodeUrlEncodedResponseInterceptor@623a8092, software.amazon.awssdk.services.s3.internal.handlers.CreateBucketInterceptor@a1cdc6d, software.amazon.awssdk.services.s3.internal.handlers.CopySourceInterceptor@175b9425]
2021-09-22 15:21:30,349 (main) DEBUG  s.a.a.c.i.ExecutionInterceptorChain:84 - Interceptor 'software.amazon.awssdk.services.s3.internal.handlers.EndpointAddressInterceptor@6ff29830' modified the message with its modifyHttpRequest method.
2021-09-22 15:21:30,350 (main) DEBUG  s.a.a.request:84 - Sending Request: DefaultSdkHttpFullRequest(httpMethod=POST, protocol=https, host=dlp-events-test-us-west-2.s3.us-west-2.amazonaws.com, port=443, encodedPath=, headers=[amz-sdk-invocation-id, Content-Length, Content-Type, User-Agent], queryParameters=[delete])
2021-09-22 15:21:30,363 (main) DEBUG  s.a.a.a.s.Aws4Signer:84 - AWS4 String to sign: AWS4-HMAC-SHA256
20210922T152130Z
20210922/us-west-2/s3/aws4_request
241396d34cf950019b63fd5535e08bd5e472c5405cca43449579916e5e766f9a
2021-09-22 15:21:30,505 (main) DEBUG  s.a.a.request:84 - Received error response: software.amazon.awssdk.services.s3.model.S3Exception: Missing required header for this request: Content-MD5 (Service: S3, Status Code: 400, Request ID: S9G6KN27VACTRAVK, Extended Request ID: pr6CzfC10oGd9uqInq4nmzzy8933FriuBJIwveYUmGRQVF4ASG5kIS+s+TUVxScKvnSV6tvRvAI=)
2021-09-22 15:21:30,506 (main) DEBUG  s.a.a.h.a.i.n.SdkSslSocket:84 - shutting down output of dlp-events-test-us-west-2.s3.us-west-2.amazonaws.com/52.92.164.50:443
2021-09-22 15:21:30,524 (main) DEBUG  s.a.a.h.a.i.n.SdkSslSocket:84 - shutting down input of dlp-events-test-us-west-2.s3.us-west-2.amazonaws.com/52.92.164.50:443
2021-09-22 15:21:30,525 (main) DEBUG  s.a.a.h.a.i.n.SdkSslSocket:84 - closing dlp-events-test-us-west-2.s3.us-west-2.amazonaws.com/52.92.164.50:443
2021-09-22 15:21:30,564 (main) ERROR  c.c.u.d.r.s.s.ReportingS3Client:72 - Failed to delete events from s3: [dfdc784c-0ed1-4c49-a17c-668f3052cabf]
software.amazon.awssdk.services.s3.model.S3Exception: Missing required header for this request: Content-MD5 (Service: S3, Status Code: 400, Request ID: S9G6KN27VACTRAVK, Extended Request ID: pr6CzfC10oGd9uqInq4nmzzy8933FriuBJIwveYUmGRQVF4ASG5kIS+s+TUVxScKvnSV6tvRvAI=)
at software.amazon.awssdk.protocols.xml.internal.unmarshall.AwsXmlPredicatedResponseHandler.handleErrorResponse(AwsXmlPredicatedResponseHandler.java:156)
at software.amazon.awssdk.protocols.xml.internal.unmarshall.AwsXmlPredicatedResponseHandler.handleResponse(AwsXmlPredicatedResponseHandler.java:106)
at software.amazon.awssdk.protocols.xml.internal.unmarshall.AwsXmlPredicatedResponseHandler.handle(AwsXmlPredicatedResponseHandler.java:84)
at software.amazon.awssdk.protocols.xml.internal.unmarshall.AwsXmlPredicatedResponseHandler.handle(AwsXmlPredicatedResponseHandler.java:42)
at software.amazon.awssdk.awscore.client.handler.AwsSyncClientHandler$Crc32ValidationResponseHandler.handle(AwsSyncClientHandler.java:95)
at software.amazon.awssdk.core.internal.handler.BaseClientHandler.lambda$successTransformationResponseHandler$6(BaseClientHandler.java:232)
at software.amazon.awssdk.core.internal.http.pipeline.stages.HandleResponseStage.execute(HandleResponseStage.java:40)
at software.amazon.awssdk.core.internal.http.pipeline.stages.HandleResponseStage.execute(HandleResponseStage.java:30)
at software.amazon.awssdk.core.internal.http.pipeline.RequestPipelineBuilder$ComposingRequestPipelineStage.execute(RequestPipelineBuilder.java:206)
at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallAttemptTimeoutTrackingStage.execute(ApiCallAttemptTimeoutTrackingStage.java:73)
at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallAttemptTimeoutTrackingStage.execute(ApiCallAttemptTimeoutTrackingStage.java:42)
at software.amazon.awssdk.core.internal.http.pipeline.stages.TimeoutExceptionHandlingStage.execute(TimeoutExceptionHandlingStage.java:78)
at software.amazon.awssdk.core.internal.http.pipeline.stages.TimeoutExceptionHandlingStage.execute(TimeoutExceptionHandlingStage.java:40)
at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallAttemptMetricCollectionStage.execute(ApiCallAttemptMetricCollectionStage.java:50)
at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallAttemptMetricCollectionStage.execute(ApiCallAttemptMetricCollectionStage.java:36)
at software.amazon.awssdk.core.internal.http.pipeline.stages.RetryableStage.execute(RetryableStage.java:80)
at software.amazon.awssdk.core.internal.http.pipeline.stages.RetryableStage.execute(RetryableStage.java:36)
at software.amazon.awssdk.core.internal.http.pipeline.RequestPipelineBuilder$ComposingRequestPipelineStage.execute(RequestPipelineBuilder.java:206)
at software.amazon.awssdk.core.internal.http.StreamManagingStage.execute(StreamManagingStage.java:56)
at software.amazon.awssdk.core.internal.http.StreamManagingStage.execute(StreamManagingStage.java:36)
at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallTimeoutTrackingStage.executeWithTimer(ApiCallTimeoutTrackingStage.java:80)
at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallTimeoutTrackingStage.execute(ApiCallTimeoutTrackingStage.java:60)
at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallTimeoutTrackingStage.execute(ApiCallTimeoutTrackingStage.java:42)
at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallMetricCollectionStage.execute(ApiCallMetricCollectionStage.java:48)
at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallMetricCollectionStage.execute(ApiCallMetricCollectionStage.java:31)
at software.amazon.awssdk.core.internal.http.pipeline.RequestPipelineBuilder$ComposingRequestPipelineStage.execute(RequestPipelineBuilder.java:206)
at software.amazon.awssdk.core.internal.http.pipeline.RequestPipelineBuilder$ComposingRequestPipelineStage.execute(RequestPipelineBuilder.java:206)
at software.amazon.awssdk.core.internal.http.pipeline.stages.ExecutionFailureExceptionReportingStage.execute(ExecutionFailureExceptionReportingStage.java:37)
at software.amazon.awssdk.core.internal.http.pipeline.stages.ExecutionFailureExceptionReportingStage.execute(ExecutionFailureExceptionReportingStage.java:26)
at software.amazon.awssdk.core.internal.http.AmazonSyncHttpClient$RequestExecutionBuilderImpl.execute(AmazonSyncHttpClient.java:193)
at software.amazon.awssdk.core.internal.handler.BaseSyncClientHandler.invoke(BaseSyncClientHandler.java:103)
at software.amazon.awssdk.core.internal.handler.BaseSyncClientHandler.doExecute(BaseSyncClientHandler.java:167)
at software.amazon.awssdk.core.internal.handler.BaseSyncClientHandler.lambda$execute$1(BaseSyncClientHandler.java:82)
at software.amazon.awssdk.core.internal.handler.BaseSyncClientHandler.measureApiCallSuccess(BaseSyncClientHandler.java:175)
at software.amazon.awssdk.core.internal.handler.BaseSyncClientHandler.execute(BaseSyncClientHandler.java:76)
at software.amazon.awssdk.core.client.handler.SdkSyncClientHandler.execute(SdkSyncClientHandler.java:45)
at software.amazon.awssdk.awscore.client.handler.AwsSyncClientHandler.execute(AwsSyncClientHandler.java:56)
at software.amazon.awssdk.services.s3.DefaultS3Client.deleteObjects(DefaultS3Client.java:2627)
at com.cisco.umbrella.delphi.reporting.save.s3.ReportingS3Client.batchDeleteEventsFromS3(ReportingS3Client.java:70)
at com.cisco.umbrella.delphi.reporting.save.s3.ReportingS3Client.deleteEventsFromS3(ReportingS3Client.java:43)
at com.cisco.umbrella.delphi.reporting.save.Handler.handleRequest(Handler.java:53)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at lambdainternal.EventHandlerLoader$PojoMethodRequestHandler.handleRequest(EventHandlerLoader.java:263)
at lambdainternal.EventHandlerLoader$PojoHandlerAsStreamHandler.handleRequest(EventHandlerLoader.java:180)
at lambdainternal.EventHandlerLoader$2.call(EventHandlerLoader.java:902)
at lambdainternal.AWSLambda.startRuntime(AWSLambda.java:340)
at lambdainternal.AWSLambda.<clinit>(AWSLambda.java:63)
at java.lang.Class.forName0(Native Method)
at java.lang.Class.forName(Class.java:348)
at lambdainternal.LambdaRTEntry.main(LambdaRTEntry.java:150)
"Missing required header for this request: Content-MD5 (Service: S3, Status Code: 400, Request ID: S9G6KN27VACTRAVK, Extended Request ID: pr6CzfC10oGd9uqInq4nmzzy8933FriuBJIwveYUmGRQVF4ASG5kIS+s+TUVxScKvnSV6tvRvAI=): software.amazon.awssdk.services.s3.model.S3Exception
software.amazon.awssdk.services.s3.model.S3Exception: Missing required header for this request: Content-MD5 (Service: S3, Status Code: 400, Request ID: S9G6KN27VACTRAVK, Extended Request ID: pr6CzfC10oGd9uqInq4nmzzy8933FriuBJIwveYUmGRQVF4ASG5kIS+s+TUVxScKvnSV6tvRvAI=)
	at software.amazon.awssdk.protocols.xml.internal.unmarshall.AwsXmlPredicatedResponseHandler.handleErrorResponse(AwsXmlPredicatedResponseHandler.java:156)
	at software.amazon.awssdk.protocols.xml.internal.unmarshall.AwsXmlPredicatedResponseHandler.handleResponse(AwsXmlPredicatedResponseHandler.java:106)
	at software.amazon.awssdk.protocols.xml.internal.unmarshall.AwsXmlPredicatedResponseHandler.handle(AwsXmlPredicatedResponseHandler.java:84)
	at software.amazon.awssdk.protocols.xml.internal.unmarshall.AwsXmlPredicatedResponseHandler.handle(AwsXmlPredicatedResponseHandler.java:42)
	at software.amazon.awssdk.awscore.client.handler.AwsSyncClientHandler$Crc32ValidationResponseHandler.handle(AwsSyncClientHandler.java:95)
	at software.amazon.awssdk.core.internal.handler.BaseClientHandler.lambda$successTransformationResponseHandler$6(BaseClientHandler.java:232)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.HandleResponseStage.execute(HandleResponseStage.java:40)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.HandleResponseStage.execute(HandleResponseStage.java:30)
	at software.amazon.awssdk.core.internal.http.pipeline.RequestPipelineBuilder$ComposingRequestPipelineStage.execute(RequestPipelineBuilder.java:206)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallAttemptTimeoutTrackingStage.execute(ApiCallAttemptTimeoutTrackingStage.java:73)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallAttemptTimeoutTrackingStage.execute(ApiCallAttemptTimeoutTrackingStage.java:42)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.TimeoutExceptionHandlingStage.execute(TimeoutExceptionHandlingStage.java:78)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.TimeoutExceptionHandlingStage.execute(TimeoutExceptionHandlingStage.java:40)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallAttemptMetricCollectionStage.execute(ApiCallAttemptMetricCollectionStage.java:50)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallAttemptMetricCollectionStage.execute(ApiCallAttemptMetricCollectionStage.java:36)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.RetryableStage.execute(RetryableStage.java:80)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.RetryableStage.execute(RetryableStage.java:36)
	at software.amazon.awssdk.core.internal.http.pipeline.RequestPipelineBuilder$ComposingRequestPipelineStage.execute(RequestPipelineBuilder.java:206)
	at software.amazon.awssdk.core.internal.http.StreamManagingStage.execute(StreamManagingStage.java:56)
	at software.amazon.awssdk.core.internal.http.StreamManagingStage.execute(StreamManagingStage.java:36)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallTimeoutTrackingStage.executeWithTimer(ApiCallTimeoutTrackingStage.java:80)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallTimeoutTrackingStage.execute(ApiCallTimeoutTrackingStage.java:60)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallTimeoutTrackingStage.execute(ApiCallTimeoutTrackingStage.java:42)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallMetricCollectionStage.execute(ApiCallMetricCollectionStage.java:48)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.ApiCallMetricCollectionStage.execute(ApiCallMetricCollectionStage.java:31)
	at software.amazon.awssdk.core.internal.http.pipeline.RequestPipelineBuilder$ComposingRequestPipelineStage.execute(RequestPipelineBuilder.java:206)
	at software.amazon.awssdk.core.internal.http.pipeline.RequestPipelineBuilder$ComposingRequestPipelineStage.execute(RequestPipelineBuilder.java:206)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.ExecutionFailureExceptionReportingStage.execute(ExecutionFailureExceptionReportingStage.java:37)
	at software.amazon.awssdk.core.internal.http.pipeline.stages.ExecutionFailureExceptionReportingStage.execute(ExecutionFailureExceptionReportingStage.java:26)
	at software.amazon.awssdk.core.internal.http.AmazonSyncHttpClient$RequestExecutionBuilderImpl.execute(AmazonSyncHttpClient.java:193)
	at software.amazon.awssdk.core.internal.handler.BaseSyncClientHandler.invoke(BaseSyncClientHandler.java:103)
	at software.amazon.awssdk.core.internal.handler.BaseSyncClientHandler.doExecute(BaseSyncClientHandler.java:167)
	at software.amazon.awssdk.core.internal.handler.BaseSyncClientHandler.lambda$execute$1(BaseSyncClientHandler.java:82)
	at software.amazon.awssdk.core.internal.handler.BaseSyncClientHandler.measureApiCallSuccess(BaseSyncClientHandler.java:175)
	at software.amazon.awssdk.core.internal.handler.BaseSyncClientHandler.execute(BaseSyncClientHandler.java:76)
	at software.amazon.awssdk.core.client.handler.SdkSyncClientHandler.execute(SdkSyncClientHandler.java:45)
	at software.amazon.awssdk.awscore.client.handler.AwsSyncClientHandler.execute(AwsSyncClientHandler.java:56)
	at software.amazon.awssdk.services.s3.DefaultS3Client.deleteObjects(DefaultS3Client.java:2627)
	at com.cisco.umbrella.delphi.reporting.save.s3.ReportingS3Client.batchDeleteEventsFromS3(ReportingS3Client.java:70)
	at com.cisco.umbrella.delphi.reporting.save.s3.ReportingS3Client.deleteEventsFromS3(ReportingS3Client.java:43)
	at com.cisco.umbrella.delphi.reporting.save.Handler.handleRequest(Handler.java:53)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)"
END RequestId: 216a70d9-cf91-5c33-81e8-291c83a64ff0
"REPORT RequestId: 216a70d9-cf91-5c33-81e8-291c83a64ff0	Duration: 1586.35 ms	Billed Duration: 1587 ms	Memory Size: 512 MB	Max Memory Used: 178 MB"
```

## Steps to Reproduce
1. Create a Lambda (runtime should be `java8`) that should delete files from S3
2. Use `deleteObjects` in order to delete a few objects from s3

## Possible Solution
N/A

## Context
We're trying to delete a few objects from S3 using a Lambda, but due to this error we're not able do delete the objects. The strange thing is once I'm running the code locally (using java), the files are getting deleted as expected (the `Content-MD5` header is being added):

### Local logs:
```
2021-09-23 11:57:54,009 (main) DEBUG  s.a.a.c.i.ExecutionInterceptorChain:84 - Creating an interceptor chain that will apply interceptors in the following order: [software.amazon.awssdk.core.internal.interceptor.HttpChecksumRequiredInterceptor@27eb3298, software.amazon.awssdk.awscore.eventstream.EventStreamInitialRequestInterceptor@200a26bc, software.amazon.awssdk.awscore.interceptor.HelpfulUnknownHostExceptionInterceptor@bc57b40, software.amazon.awssdk.services.s3.internal.handlers.EnableChunkedEncodingInterceptor@1b5bc39d, software.amazon.awssdk.services.s3.internal.handlers.DisableDoubleUrlEncodingInterceptor@655a5d9c, software.amazon.awssdk.services.s3.internal.handlers.EnableTrailingChecksumInterceptor@1494b84d, software.amazon.awssdk.services.s3.internal.handlers.CreateMultipartUploadRequestInterceptor@34abdee4, software.amazon.awssdk.services.s3.internal.handlers.GetObjectInterceptor@71a9b4c7, software.amazon.awssdk.services.s3.internal.handlers.ExceptionTranslationInterceptor@4628b1d3, software.amazon.awssdk.services.s3.internal.handlers.AsyncChecksumValidationInterceptor@77cf3f8b, software.amazon.awssdk.services.s3.internal.handlers.EndpointAddressInterceptor@1df98368, software.amazon.awssdk.services.s3.internal.handlers.SignerOverrideInterceptor@21ca139c, software.amazon.awssdk.services.s3.internal.handlers.GetBucketPolicyInterceptor@226f885f, software.amazon.awssdk.services.s3.internal.handlers.PutObjectInterceptor@2cd2c8fe, software.amazon.awssdk.services.s3.internal.handlers.SyncChecksumValidationInterceptor@7d61eccf, software.amazon.awssdk.services.s3.internal.handlers.DecodeUrlEncodedResponseInterceptor@cc6460c, software.amazon.awssdk.services.s3.internal.handlers.CreateBucketInterceptor@52350abb, software.amazon.awssdk.services.s3.internal.handlers.CopySourceInterceptor@681aad3b]
2021-09-23 11:57:54,018 (main) DEBUG  s.a.a.c.i.ExecutionInterceptorChain:84 - Interceptor 'software.amazon.awssdk.core.internal.interceptor.HttpChecksumRequiredInterceptor@27eb3298' modified the message with its modifyHttpRequest method.
2021-09-23 11:57:54,019 (main) DEBUG  s.a.a.c.i.ExecutionInterceptorChain:84 - Interceptor 'software.amazon.awssdk.services.s3.internal.handlers.EndpointAddressInterceptor@1df98368' modified the message with its modifyHttpRequest method.
2021-09-23 11:57:54,019 (main) DEBUG  s.a.a.request:84 - Sending Request: DefaultSdkHttpFullRequest(httpMethod=POST, protocol=https, host=dlp-events-test-us-west-2.s3.us-west-2.amazonaws.com, port=443, encodedPath=, headers=[amz-sdk-invocation-id, Content-Length, Content-MD5, Content-Type, User-Agent], queryParameters=[delete])
2021-09-23 11:57:54,020 (main) DEBUG  s.a.a.a.s.Aws4Signer:84 - AWS4 String to sign: AWS4-HMAC-SHA256
20210923T085754Z
20210923/us-west-2/s3/aws4_request
e296c2a5c981df475db78d40166ecab4027a72531e5788a53edf43c4719dfe2f
2021-09-23 11:57:54,351 (main) DEBUG  s.a.a.request:84 - Received successful response: 200
2021-09-23 11:57:54,357 (main) DEBUG  s.a.a.h.a.i.n.SdkSslSocket:84 - shutting down output of dlp-events-test-us-west-2.s3.us-west-2.amazonaws.com/52.218.230.41:443
2021-09-23 11:57:54,358 (main) DEBUG  s.a.a.h.a.i.n.SdkSslSocket:84 - closing dlp-events-test-us-west-2.s3.us-west-2.amazonaws.com/52.218.230.41:443
```

## Your Environment

* AWS Java SDK version used: `Java 8 on Amazon Linux 1`
* JDK version used: `2.17.42`
* Operating System and version: AWS Lambda Java 8

**Comment by gshondal** [comment]
Hello  @baradm100,

While we do not maintain the AWS Lambda environment, this information has been forwarded to the internal Lambda service team on your behalf.

**Comment by debora-ito** [comment]
@baradm100 thank you for the detailed report.

In fact, the behavior you've described is due to namespace conflict of `execution.interceptors` files, reported here: https://github.com/aws/aws-sdk-java-v2/issues/2676. 

The Content-MD5 is calculated by one of the interceptors, and if you use an Uber jar in Lambda this interceptor may be getting overwritten. We have a PR to fix the situation: https://github.com/aws/aws-sdk-java-v2/pull/2734. Will. be available in the next release.

**Comment by debora-ito** [comment]
The fix for the namespace conflict was released in SDK version `2.17.46`. 

Please upgrade it and let us know if you still see the error.
